sklearn/ensemble/voting.py

```diff
@@ -339,9 +369,157 @@ def get_params(self, deep=True):
             Setting it to True gets the various classifiers and the parameters
             of the classifiers as well
         """
-        return super(VotingClassifier,
-                     self)._get_params('estimators', deep=deep)
+        return self._get_params('estimators', deep=deep)
 
-    def _predict(self, X):
-        """Collect results from clf.predict calls. """
-        return np.asarray([clf.predict(X) for clf in self.estimators_]).T
+
+class AverageRegressor(BaseVoting, RegressorMixin, TransformerMixin):
+    """
```

Comment:
PEP257: this line should contain a 1-line summary of what the class is, followed by a blank line.